src/reducer.test.js

```diff
@@ -63,49 +64,53 @@ describe('reducer', () => {
         restaurants: [],
       };
 
-      const restaurants = [
-        { id: 1, name: '마법사주방' },
-      ];
+      const state = reducer(initialState, setRestaurants(RESTAURANTS));
 
-      const state = reducer(initialState, setRestaurants(restaurants));
-
-      expect(state.restaurants).toHaveLength(1);
+      expect(state.restaurants).toHaveLength(RESTAURANTS.length);
     });
   });
 
   describe('selectRegion', () => {
     it('changes selected region', () => {
       const initialState = {
-        regions: [
-          { id: 1, name: '서울' },
-        ],
+        regions: REGIONS,
         selectedRegion: null,
       };
 
-      const state = reducer(initialState, selectRegion(1));
+      const targetId = 1;
+
+      const state = reducer(initialState, selectRegion(targetId));
 
-      expect(state.selectedRegion).toEqual({
-        id: 1,
-        name: '서울',
-      });
+      expect(state.selectedRegion).toEqual(REGIONS.find(equal('id', targetId)));
```

Comment:
find를 사용하는 것보다 데이터 자체를 넣어주는건 어떠신가요?

Reply:
```suggestion
      expect(state.selectedRegion).toEqual(REGIONS[0]);
```

이런식으로 직접 넣어주는게 좋을까요??🤔